action.yml

```diff
@@ -11,6 +11,10 @@ inputs:
   access-token:
     description: 'access token with repo scope'
     required: true
+  github-api-url:
+    description: 'url for the github API'
+    required: false
+    default: 'https://api.github.com'
```

Comment:
You could use the environment variable `GITHUB_API_URL` for the default. The variable is set for enterprise servers.